microsoft/runbook/my.yml

```diff
@@ -0,0 +1,58 @@
+name: azure default
```

Comment:
This is just for testing purpose. Will remove it once incorporate all the review comments